core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -122,6 +173,43 @@ interface DateTimeFormatHandler {
           .put("%x", (date) -> null)
           .build();
 
+  private static final Map<String, StrToDateHandler> STR_TO_DATE_HANDLERS =
```

Comment:
Why not `Map<String, String>`?

Reply:
Originally this was based off of the implementation for `date_format` and `time_format`, and I did it this way because I thought I would need to have specific implementations for some corner cases. I don't think it's necessary anymore though, so I will try to change the implementation.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/231/commits/fe86d6156c2fb952bfa9feecac36a4a240f6ea2d